script.js

```diff
@@ -1,4 +1,479 @@
+    outputText += `<br> The dealer
+    s face-down card is ${computerCards[numOfComputerCards - 1].name} of ${
+      computerCards[numOfComputerCards - 1].suit
+    }. <br> The dealer's cards total ${computerCardsTotal}. <br><br> You win! <br> Your points: ${playerBankroll} <br> The dealer's points: ${computerBankroll}. <br><br> Press submit.`;
+    gameMode = GAME_MODE_BETTING;
+  }
+  // when the player's cards and computer's cards totals are the same, it is a tie
+  else if (computerCardsTotal == playerCardsTotal && computerCardsTotal <= 21) {
+    outputText += `<br> The dealer
+    s face-down card is ${computerCards[numOfComputerCards - 1].name} of ${
+      computerCards[numOfComputerCards - 1].suit
+    }. <br> The dealer's cards total ${computerCardsTotal}. <br><br> It's a tie! <br><br> Your points: ${playerBankroll} <br> The dealer's points: ${computerBankroll}.<br><br> Press submit.`;
+    gameMode = GAME_MODE_BETTING;
+  } else {
+    outputText += `ERROR!`;
+  }
+
+  return outputText;
+};
+
+/**
+ * Create a standard 52-card deck
+ */
+var makeDeck = function () {
+  // Initialise an empty deck array
+  var cardDeck = [];
+  // Initialise an array of the 4 suits in our deck. We will loop over this array.
+  var suits = ["hearts", "diamonds", "clubs", "spades"];
+
+  // Loop over the suits array
+  var suitIndex = 0;
+  while (suitIndex < suits.length) {
+    // Store the current suit in a variable
+    var currentSuit = suits[suitIndex];
+
+    // Loop from 1 to 13 to create all cards for a given suit
+    // Notice rankCounter starts at 1 and not 0, and ends at 13 and not 12.
+    // This is an example of a loop without an array.
+    var rankCounter = 1;
+    while (rankCounter <= 13) {
+      // By default, the card name is the same as rankCounter
+      var cardName = rankCounter;
+
+      // If rank is 1, 11, 12, or 13, set cardName to the ace or face card's name
+      if (cardName == 1) {
+        cardName = "ace";
+      } else if (cardName == 11) {
+        cardName = "jack";
+      } else if (cardName == 12) {
+        cardName = "queen";
+      } else if (cardName == 13) {
+        cardName = "king";
+      }
+
+      // Create a new card with the current name, suit, and rank
+      var card = {
+        name: cardName,
+        suit: currentSuit,
+        rank: rankCounter,
+      };
+
+      // Add the new card to the deck
+      cardDeck.push(card);
+
+      // Increment rankCounter to iterate over the next rank
+      rankCounter += 1;
+    }
+
+    // Increment the suit index to iterate over the next suit
+    suitIndex += 1;
+  }
+
+  // If card is jack, queen or king, set rank to be 10 for game of Blackjack
+  cardDeck[10].rank = 10;
+  cardDeck[11].rank = 10;
+  cardDeck[12].rank = 10;
+  cardDeck[23].rank = 10;
+  cardDeck[24].rank = 10;
+  cardDeck[25].rank = 10;
+  cardDeck[36].rank = 10;
+  cardDeck[37].rank = 10;
+  cardDeck[38].rank = 10;
+  cardDeck[49].rank = 10;
+  cardDeck[50].rank = 10;
```

Comment:
generally, it is not desirable to have brute-forced code in our program as it makes it open to a greater probability of errors. Perhaps you can add a new variable in your while loop in line 392 that keeps track of the cardRank. So if rankCounter<=10, cardRank=rankCounter and if rankCounter>10, you can set cardRank to 10 in your if and else if statements in line 397 onwards